http.md

```diff
@@ -227,7 +252,8 @@ pass parameters like what port to use.
 
   `http --auth Pirate:memelord42`
 
-    As in the first example, but require all clients to log in with the username "Pirate" and password "memelord42".
```

Comment:
Why we splitting these lines, and why at such random point? You're not splitting other long lines.

Reply:
Fixed